flexmeasures/utils/coding_utils.py

```diff
@@ -69,21 +69,38 @@ def sort_dict(unsorted_dict: dict) -> dict:
     return sorted_dict
 
 
+# The parsing functionality for sensors_to_show in flatten_unique will be moved to SensorsToShowSchema.
```

Comment:
Yes, correct, please go ahead with that. This kind of logic could be part of the deserialization function in the schema.

The only clean solution I see is to use that function to create from input that is diverse (all supported formats) the most informative format for each entry, which is `{"title": "row2 Title", "sensors": [41, 42]}`. So for instance, `44` becomes `{"title": None, "sensors": [44]}`.

Then, all code dealing with this after deserialization can now peacefully use that format and not worry if others are in there (no if/else blocks).

The code in `chart_for_multiple_sensors` which just needs a list of sensors can get them rather quickly this way as well, I believe. Just extend the list which each "sensors" list.

Reply:
this comment was addressed in code changes.